sal_interfaces/MTMount/MTMount_Events.xml

```diff
@@ -1,85 +1,245 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
 <SALEventSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <Enumeration>
+        ActuatorState_Disabled,
+        ActuatorState_MovingPointToPoint,
+        ActuatorState_TrackingOrSlewing,
+        ActuatorState_Stopping,
+        ActuatorState_Stopped
+    </Enumeration>
+  <Enumeration>
+        Subsystem_AZIMUTH_AXIS=0x0001,
+        Subsystem_AZIMUTH_CABLE_WRAP=0x0002,
+        Subsystem_AZIMUTH_THERMAL=0x0004,
+        Subsystem_BALANCE=0x0008,
+        Subsystem_CABINET=0x0010,
+        Subsystem_CAMERA_CABLE_WRAP=0x0020,
+        Subsystem_DEPLOYABLE_PLATFORM=0x0040,
+        Subsystem_ELEVATION_AXIS=0x0080,
+        Subsystem_ELEVATION_THERMAL=0x0100,
+        Subsystem_LOCKING_PINS=0x0200,
+        Subsystem_MIRROR_COVER=0x0400,
+        Subsystem_MIRROR_COVER_LOCK=0x0800,
+        Subsystem_MAIN_POWER_SUPPLY=0x1000,
+        Subsystem_OIL_SUPPLY_SYSTEM=0x2000,
+        Subsystem_TOP_END_CHILLER=0x4000
+    </Enumeration>
+  <Enumeration>
+        AxisState_Unknown=0,
+        AxisState_Off=1,
+        AxisState_Enabled=2,
+        AxisState_Fault=3
+    </Enumeration>
+  <Enumeration>
+        DriveState_Unknown=0,
+        DriveState_Off=1,
+        DriveState_Moving=2,
+        DriveState_Stopping=3,
+        DriveState_Stopped=4,
+        DriveState_Fault=5
+    </Enumeration>
   <SALEvent>
     <Subsystem>MTMount</Subsystem>
-    <Version>3.2.0</Version>
-    <Author>Ander Ansuategi</Author>
-    <EFDB_Topic>MTMount_logevent_mountState</EFDB_Topic>
-    <Alias>mountState</Alias>
-    <Explanation>http://sal.lsst.org/SAL/Events/MTMount_mountState.html</Explanation>
-    <item>
-      <EFDB_Name>id</EFDB_Name>
-      <Description>Identifier of the state</Description>
+    <EFDB_Topic>MTMount_logevent_axesInPosition</EFDB_Topic>
+    <Description>Are the elevation and azimuth axes in position? If tracking, in position means the slew is done and the axes are now tracking the target.</Description>
+    <item>
+      <EFDB_Name>elevation</EFDB_Name>
+      <Description>Is the elevation axis in position?</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>azimuth</EFDB_Name>
+      <Description>Is the azimuth axis in position?</Description>
+      <IDL_Type>boolean</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALEvent>
+  <SALEvent>
+    <Subsystem>MTMount</Subsystem>
+    <EFDB_Topic>MTMount_logevent_axesState</EFDB_Topic>
+    <Description>State of the elevation and azimuth axes.</Description>
+    <item>
+      <EFDB_Name>elevation</EFDB_Name>
+      <Description>Elevation state, as an ActuatorState constant enum</Description>
```

Comment:
Nitpicking but please add a trailing `.` to terminate the phrase. Same below which I assume happened due to copying and pasting.

Reply:
I did a global (with MTMount) correction of all Descriptions that did not end with . or ?